README.md

```diff
@@ -197,8 +116,8 @@ Build the image locally using this git repository as the [build context](https:/
 ```console
 docker build \
   --build-arg VERSION=0.0.1 \
```

Comment:
The `VERSION` arg needs to be updated based on how https://github.com/cisagov/li-pca-app-docker/pull/7/files#r802964830 turns out.